xds/src/test/java/io/grpc/xds/internal/security/CommonTlsContextTestsUtil.java

```diff
@@ -48,10 +48,14 @@ public class CommonTlsContextTestsUtil {
   public static final String SERVER_0_PEM_FILE = "server0.pem";
   public static final String SERVER_0_KEY_FILE = "server0.key";
   public static final String SERVER_1_PEM_FILE = "server1.pem";
+  public static final String SERVER_1_SPIFFE_PEM_FILE = "server1_spiffe.pem";
   public static final String SERVER_1_KEY_FILE = "server1.key";
   public static final String CLIENT_PEM_FILE = "client.pem";
+  public static final String CLIENT_SPIFFE_PEM_FILE = "client_spiffe.pem";
   public static final String CLIENT_KEY_FILE = "client.key";
   public static final String CA_PEM_FILE = "ca.pem";
+  public static final String SPIFFE_TRUST_BUNDLE_FILE = "spiffebundle.json";
```

Comment:
s/spiffebundle/spiffe_trust_map

Similarly below.

Reply:
Do you also want to rename all the files? The `spiffebundle_X` pattern is what we used in previous Spiffe PRs - https://github.com/grpc/grpc-java/tree/master/core/src/test/resources/io/grpc/internal